InstagramStories/Source/StoryPreview/IGStoryPreviewController.swift

```diff
@@ -16,7 +16,7 @@ import UIKit
 final class IGStoryPreviewController: UIViewController, UIGestureRecognizerDelegate {
     
     //MARK: - Private Vars
-    private var _view: IGStoryPreviewView {return view as! IGStoryPreviewView}
+    private var _view: IGStoryPreviewView = IGStoryPreviewView()
```

Comment:
Unnecessary instance creation for _view. So could you please change it to `private var _view: IGStoryPreviewView!` because in loadView anyway we are creating the **IGStoryPreviewView** instance with help of layoutType.